packages/sync/src/config/SyncConfig.ts

```diff
@@ -39,7 +38,11 @@ export class SyncConfig implements DataSyncConfig {
     }
   }
 
-  public storage?: PersistentStore<PersistedData>;
+  public storage?: LocalForage;
+  public storageOptions = {
+    name: "applicationDB",
```

Comment:
Having it this way gives no benefit over local storage. 
Do you think we can try different `driver` option?

See: https://github.com/localForage/localForage

Reply:
@wtrocki you can see the default order here: https://github.com/localForage/localForage/blob/master/src/localforage.js#L23-L27

it uses IndexedDB first